files/tasks.py

```
@@ -1100,13 +1184,21 @@ def task_sent_handler(sender=None, headers=None, body=None, **kwargs):
 def kill_ffmpeg_process(filepath):
     # this is not ideal, ffmpeg pid could be linked to the Encoding object
-    cmd = "ps aux|grep 'ffmpeg'|grep %s|grep -v grep |awk '{print $2}'" % filepath
-    result = subprocess.run(cmd, stdout=subprocess.PIPE, shell=True)
-    pid = result.stdout.decode("utf-8").strip()
-    if pid:
-        cmd = "kill -9 %s" % pid
-        result = subprocess.run(cmd, stdout=subprocess.PIPE, shell=True)
-    return result
+    try:
+        # Use pgrep to find ffmpeg processes with the filepath
+        result = subprocess.run(
```

Ruff (0.13.3)

`subprocess` call: check for execution of untrusted input

(S603)